### PR TITLE
source-jotform contribution from topefolorunso

### PR DESCRIPTION
**Pull request by topefolorunso**
Auto-generated PR by the Connector Builder for source-jotform

### CODE REVIEW

File: docs/integrations/sources/jotform.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Jotform
+Jotform is a powerful online form builder that makes it easy to create robust forms and collect important data.
+
```
Comment (natikgadzhi) [inline docs/integrations/sources/jotform.md:3]: @topefolorunso can you please add a little snippet explaining how to obtain the API key?

**Review by natikgadzhi — COMMENTED** [review]
A couple of docs improvements to make.

File: docs/integrations/sources/jotform.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+## Configuration
+
+| Input | Type | Description | Default Value |
+|-------|------|-------------|---------------|
+| `api_key` | `string` | API Key.  |  |
+| `end_date` | `string` | End date.  |  |
+| `start_date` | `string` | Start date.  |  |
```
[the remark this answers is not in the document]
Comment (topefolorunso) [reply docs/integrations/sources/jotform.md]: Fixed @natikgadzhi

File: airbyte-integrations/connectors/source-jotform/metadata.yaml [hunk 3]
```diff
@@ -0,0 +1,35 @@
+metadataSpecVersion: "1.0"
+data:
+  allowedHosts:
+    hosts:
+      - "*"
```
Comment (natikgadzhi) [inline airbyte-integrations/connectors/source-jotform/metadata.yaml:5]: Please add the api host that's actually being used here.

**Review by natikgadzhi — COMMENTED** [review]
Metadata needs the api host in allowedHosts. We will infer it automatically when you make a PR soon, but for now, you'd have to do it manually on all your new connector PRs. @topefolorunso

File: airbyte-integrations/connectors/source-jotform/metadata.yaml (hunk 3 above)
Comment (topefolorunso) [reply airbyte-integrations/connectors/source-jotform/metadata.yaml]: Fixed @natikgadzhi

**Comment by natikgadzhi** [comment]
Good work @topefolorunso! 👏🏼

**Pull request merged**